### PR TITLE
SCSS refactor

### PR DESCRIPTION
**Pull request by simplyluke**
Compile to single css file - should be faster - better organization

### CODE REVIEW

File: static/js/components/CardComponent.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
 
-require('../../css/CardComponent.scss');
+// require('../../css/CardComponent.scss');
```
Comment (clairebingham) [inline static/js/components/CardComponent.js]: Can these lines just be deleted?

File: static/js/components/CardGrid.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from "react";
 import CardComponent from "./CardComponent"
 
-require('../../css/CardComponent.scss');
+// require('../../css/CardComponent.scss');
```
Comment (clairebingham) [inline static/js/components/CardGrid.js]: Same with this line, can it just be deleted?

Comment (simplyluke) [reply static/js/components/CardGrid.js]: Yes, will fix

**Comment by clairebingham** [comment]
👍

**Comment by clairebingham** [comment]
We should def have Liam take a look at this too though

**Pull request merged**